app/lib/filters/show_all_applications.rb

```diff
@@ -0,0 +1,25 @@
+module Filters
+  class ShowAllApplications < Base
+    def apply
+      unless show_all_applications.include?("show_all")
+        ninety_days_ago = 90.days.ago
+        new_scope =
+          scope.where(
+            "(awarded_at >= :ninety_days OR awarded_at IS NULL)
+          AND (declined_at >= :ninety_days OR declined_at IS NULL)
+          AND (withdrawn_at >= :ninety_days OR withdrawn_at IS NULL)",
+            ninety_days: ninety_days_ago,
+          )
+        return new_scope
+      end
+
+      scope
+    end
+
+    private
+
+    def show_all_applications
+      Array(params[:display]).compact_blank
+    end
```

Comment:
Following on from my suggestion about renaming the check box name.

```suggestion
    def show_all?
      ActiveModel::Type::Boolean.new.cast(params[:show_all])
    end
```

`ActiveModel::Type::Boolean.new.cast` parses the string for us so the string `"true"` is treated as a `true` boolean.